linode/objbucket/resource.go

```diff
@@ -499,6 +595,9 @@ func matchRulesWithSchema(rules []*s3.LifecycleRule, declaredRules []interface{}
 
 	// populate remaining values
 	for _, rule := range ruleMap {
+		tflog.Debug(ctx, "adding new rules", map[string]any{
+			"rule": rule,
+		})
```

Comment:
Given this helper is for reconciling the order of rules at read-time, it might make sense to give this a trace severity